examples/dreamcast/libdream/keyboard/keyboard.c

```diff
@@ -1,9 +1,11 @@
 #include <kos.h>
 
-void kb_test(void) {
+static int x = 20, y = 20 + 24;
```

Comment:
Why making this static? 
We can pass these variables as parameters to a function void kb_test(...), also, 20 and 24 can be named key values.
Like KB_KEY_D or something like that.

Reply:
Sorry for the late response. Getting back to KOS work now.

The values are really just for the starting x,y coordinates on the screen for where we're drawing text. So it starts out with margins of 20x20px... I think the 24 comes from the height of one font character?

At very least, these can be constants for the margins and then use the font height...

Knowing what these magic numbers represent, do you still think they should get passed in? I guess `main()` could pass them by pointer and be the one initializing and maintaining them across calls...